report-generator/src/test/java/com/koval/jresolver/report/ReportGeneratorTests.java

```diff
@@ -0,0 +1,114 @@
+package com.koval.jresolver.report;
+
+import java.io.File;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.MessageDigest;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+import org.apache.uima.pear.util.FileUtil;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.koval.jresolver.classifier.configuration.ClassifierProperties;
+import com.koval.jresolver.classifier.core.Classifier;
+import com.koval.jresolver.classifier.core.impl.DocClassifier;
+import com.koval.jresolver.classifier.results.ClassifierResult;
+import com.koval.jresolver.connector.bean.JiraIssue;
+import com.koval.jresolver.report.core.ReportGenerator;
+import com.koval.jresolver.report.core.impl.HtmlReportGenerator;
+import com.koval.jresolver.report.results.TotalResult;
+import com.koval.jresolver.rules.core.impl.DroolsRuleEngine;
+import com.koval.jresolver.rules.results.RulesResult;
+
+
+public class ReportGeneratorTests {
+
+    private ReportGenerator reportGenerator;
+
+    @Before
+    public void setUp() throws Exception {
+        ClassifierProperties classifierProperties = new ClassifierProperties("classifier.properties");
+        Classifier classifier = new DocClassifier(classifierProperties);
+        reportGenerator = new HtmlReportGenerator(classifier, new DroolsRuleEngine());
+    }
+
+    @Test
+    public void configurationTest() throws Exception {
+        boolean flag = false;
+        File file = new File("../output");
```

Comment:
Working with files in unit tests is considered bad practice.
Try to implement tests without interacting with the file system.
Look for information about mock objects and Mockito library.